shoop/admin/modules/orders/json_order_creator.py

```diff
@@ -98,8 +98,21 @@ def _process_product_line(self, source, sline, sl_kwargs):
                 "shop": source.shop
             }), code="no_shop_product"))
             return False
+        supplier = shop_product.suppliers.first()  # TODO: Allow setting a supplier?
+        is_orderable = True
+        for message in shop_product.get_orderability_errors(
+                supplier=supplier, quantity=sl_kwargs["quantity"], customer=source.customer):
+            self.add_error(ValidationError((_("Product %(product)s is not orderable: %(error)s") % {
+                "product": product,
+                "error": str(message.args[0])
+            }), code="product_not_orderable"))
```

Comment:
It does uses messages from from get_orderabilty_errors. Those messages is just like "This product is not visible" so I would like to add some product information to errors.